homeassistant/components/vegehub/strings.json

```
@@ -0,0 +1,45 @@
+{
+  "title": "VegeHub",
+  "config": {
+    "flow_title": "{host}",
+    "step": {
+      "user": {
+        "title": "Set up VegeHub",
+        "description": "Do you want to set up this VegeHub?",
+        "data": {
+          "ip_address": "Enter the IP address of target VegeHub"
```

```suggestion
          "ip_address": "IP address"
```
keep this should, long text is in description